static/app/views/insights/common/queries/useHasFirstSpan.tsx

```diff
@@ -52,6 +53,9 @@ export function useHasFirstSpan(module: ModuleName, projects?: Project[]): boole
   //  - [] empty list represents "My Projects"
   //  - [-1] represents "All Projects"
   //  - [.., ..] otherwise, represents a list of project IDs
+  if (pageFilters.selection.projects.length === 0 && isActiveSuperuser()) {
+    selectedProjects = allProjects; // when superuser is enabled, My Projects isn't applicable, and in reality all projects are selected when projects.length === 0
+  }
   if (pageFilters.selection.projects.length === 0) {
     selectedProjects = allProjects.filter(p => p.isMember);
```

Comment:
```suggestion
    selectedProjects = allProjects.filter(p => p.isMember || isActiveSuperuser());
```
same thing but shorter, maybe less obvious what it does though 🤷

Reply:
Yeah, that's a good call out, thanks! I do think it is less obvious what's going on, so I decided to leave as is for now

Reply:
At first I thought it wasn't obvious, but it makes sense actually as they are both conditions on the user (isMemeber or isSuperusuer).
It is an unnecessary filter, but the performance of that is negligible